cloudevents/bindings/amqp-protocol-binding.md

```diff
@@ -308,6 +318,7 @@ content-type: application/cloudevents+json; charset=utf-8
   Format
 - [OASIS-AMQP-1.0][oasis-amqp-1.0] OASIS Advanced Message Queuing Protocol
   (AMQP) Version 1.0
+ -[JMS20][JMS20] JSR-343 Java Message Service 2.0
```

Comment:
Swap the space and "-". The MD->HTML conversion looks funky:
![image](https://user-images.githubusercontent.com/1944671/185980246-92cab562-2dcd-4a1c-aa54-a9724758b315.png)